### ISSUE TITLE
 `create-react-app` example not working

### ISSUE THREAD

**Issue by lgenzelis** [lingui/js-lingui#976]
**Describe the bug**
After a fresh clone of the repo, I can't run the  `create-react-app` example. 

This is the result:
<img width="1078" alt="Screen Shot 2021-02-12 at 11 29 05 AM" src="https://user-images.githubusercontent.com/4582440/107780509-87883400-6d25-11eb-9c21-dc70eceb7dd8.png">

Also, terminal output shows:

```
Compiled with warnings.

./src/locales/cs/messages.po (./node_modules/@lingui/loader!./src/locales/cs/messages.po)
TypeError: Cannot read property 'locale' of null

./src/locales/en/messages.po (./node_modules/@lingui/loader!./src/locales/en/messages.po)
TypeError: Cannot read property 'locale' of null

src/App.tsx
  Line 13:9:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
```

**To Reproduce**
```
git clone git@github.com:lingui/js-lingui.git
cd js-lingui/examples/create-react-app/
yarn install
yarn start
```

**Expected behavior**
Example should work.

**Additional context**
Tried with the latest commit (30a4ba94), and also with specific tags (v3.4.0, v3.2.0, ...). Same result every time.

**Comment by semoal** [comment]
We resolved it on 3.5.x version, if you do yarn install you will see that 3.2.3 is installed.

I'll create a commit now updating all examples versions, thanks for the reminder

**Comment by lgenzelis** [comment]
Just pulled your commit. You forgot to update `@lingui/react` under dependencies to 3.5.1 as well. :) I did it, and now the example works fine. Thanks a lot for the speed with which you tackled this @semoal  🙌 🙌
